WordPress/Classes/Networking/RemoteReaderSiteInfo.h

```diff
@@ -14,5 +14,6 @@
 @property (nonatomic, strong) NSString *siteName;
 @property (nonatomic, strong) NSString *siteURL;
 @property (nonatomic, strong) NSNumber *subscriberCount;
+@property (nonatomic, strong) NSString *path;
```

Comment:
In the context of SiteInfo, `path` looks a bit ambiguous to me. Maybe something a bit more descriptive like `postsEndpoint`?